authentication/native/server.go

```diff
@@ -87,7 +81,7 @@ func (server *authServer) Validate(authToken authentication.AuthToken) error {
 }
 
 func (server *authServer) validateExpiration(token authentication.AuthToken) error {
-	block, err := server.getBlockByHash(context.Background(), token.GetBlockHash())
+	block, err := server.blockhashHandler.GetBlockByHash(context.Background(), token.GetBlockHash())
```

Comment:
because this call uses `context.Backgound()`, I suggest removing the context parameter of this method and instead save the one created on the blockHashHandler component's constructor